Source/Core/Plugins/EventParser/Raygun/RaygunEventParserPlugin.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using Exceptionless.Core.Extensions;
+using Exceptionless.Core.Models;
+using Exceptionless.Core.Pipeline;
+
+namespace Exceptionless.Core.Plugins.EventParser.Raygun {
+    [Priority(0)]
+    public class RaygunEventParserPlugin : IEventParserPlugin {
+        private readonly RaygunEventMapper _mapper = new RaygunEventMapper();
+
+        public List<PersistentEvent> ParseEvents(string input, int apiVersion, string userAgent) {
+            if (userAgent != "raygun" && apiVersion != 1)
```

Comment:
Okay 😃

But better change to using *or* (`||`) instead, to fix the current issues.

`and` expectation handled:
1. **raygun** with **v1**: 👍
2. **raygun** with **v2**: 👎
3. **other** with **v1**: 👎
4. **other** with **v2**: 👍

2 + 3 will in other words, not do `return null` as expected, but passing through.

`or` will on the other hand, cover all 4 cases above.